scripts/README.md

```diff
@@ -0,0 +1,177 @@
+
+### Dependencies
+
+Before attempting to use any update operation, please ensure all dependencies
+are correctly installed and set up in your machine.
+
+* [Registers Credentials set up](https://github.com/openregister/credentials).
+* Python 3.
+* [Pipenv](https://pipenv.readthedocs.io/en/latest/).
+
+Install the update dependencies:
+
+```sh
+pipenv install
+```
+
+## Getting started
+
+The quickest way to get a sense of what is possible is to check the update
+help:
+
+```
+pipenv run update --help
+```
+
+
+## Update operations
+
+Note that in order to get any metadata change appear on the frontend you will
+need to [update register metadata on
+registers-frontend](https://github.com/openregister/registers-frontend#updating-register-metadata).
+
+
+### Add data to a register
+
+The update CLI expects a TSV or a CSV, with a header row where each value is
+the exact field name (i.e. identifier) and the rest of rows the the data to
+upload.
+
+```sh
+pipenv update data --rsf=../registry-data/rsf/country.rsf \
+  --phase=test ./new-data.tsv
+```
+
+
+### Change the register title
+
+The name of a register is immutable because it is its unique ID. But the
+register can also have a human-friendly text, the title, which can be updated
+as follows:
+
+```sh
+pipenv run update context --rsf=../registry-data/rsf/country.rsf \
+  --phase=test title "A list of countries recognised by the UK"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update context --help
+```
+
+
+### Change the register description
+
+```sh
+pipenv run update context --rsf=../registry-data/rsf/country.rsf \
+  --phase=test description "A list of countries recognised by the UK"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update context --help
+```
+
+
+### Change the custodian name
+
+```sh
+pipenv run update context --rsf=../registry-data/rsf/country.rsf \
+  --phase=test custodian "Jane Doe"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update context --help
+```
+
+
+### Change a field description
+
+```sh
+pipenv run update schema --rsf=../registry-data/rsf/country.rsf \
+  --phase=test start-date "The date the country was constituted"
+```
+
+Check the context subcommand help for more details:
+
+```sh
+pipenv run update schema --help
+```
+
+
+## Initial load
+
+Once you have an empty ORJ register, you can do an initial load using an RSF
+file.
+
+
+Assuming there is an empty register in the test environment for the
+`web-colour` register you can upload the register by:
+
+
+```sh
+pipenv run update init --phase=test ./web-colour.rsf
+```
+
+
+## The Field and Register registers
+
+
+Instances of ORJ serving the Field and Register Registers read yaml files
+containing fields and registers from S3 on startup.
+
+There is an Ansible task in the deployment repo to generate the files in S3
+from the application. You will need to run this after reloading the metadata
+RSF.
```

Comment:
it is not only the ansible task as you also need to re-deploy the application. Better to link to: https://github.com/openregister/deployment/blob/master/docs/extra-steps-beta.md#4-run-ansible-task steps 4 and 5.